payroll-processor-functions/src/payroll-processor-functions/.vscode/extensions.json

```diff
@@ -1,6 +0,0 @@
-{
-  "recommendations": [
```

Comment:
These are defined at the workspace level now

Reply:
Yeah the way this repo started was originally 3 separate projects so i had multiple .vs and .vscode files and editorconfigs for each project, never got around to consolidating them. Thanks for the cleanup!